bridge/src/test/java/com/bridge/RenderManager/RenderManagerTest.java

```diff
@@ -0,0 +1,69 @@
+package com.bridge.renderManager;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+import com.bridge.piece.Coord;
+import com.bridge.piece.Size;
+import com.bridge.piece.render.RenderManager;
+import com.bridge.piece.sound.*;
```

Comment:
update the package import here as it has changed to "renderhandler"

Reply:
name updated